frontend/src/locales/README.md

```diff
@@ -1,4 +1,4 @@
 This folder will contain generated translations in your development or production OpenProject environment.
 
 To find out more how to contribute to translations,
-please see [our translation guide](^https://docs.openproject.org/development/#translations).
\ No newline at end of file
+please see [our translation guide](https://docs.openproject.org/development/#translations).
```

Comment:
The subdomain is no longer in use and redirects to https://www.openproject.org/docs/development/#translations